packages/graphql/src/schema/validation/custom-rules/utils/path-parser.ts

```diff
@@ -33,21 +34,34 @@ export type ObjectOrInterfaceWithExtensions =
 
 export function getPathToNode(
     path: readonly (number | string)[],
-    ancenstors: readonly (ASTNode | readonly ASTNode[])[]
+    ancestors: readonly (ASTNode | readonly ASTNode[])[]
 ): [
     Array<string>,
     ObjectOrInterfaceWithExtensions | FieldDefinitionNode | undefined,
     ObjectOrInterfaceWithExtensions | undefined
 ] {
```

Comment:
**Nothing to change now** but we may want to consider changing this function to return an object. From looking at the function declaration I have no idea what the return variables are supposed to be